### PR TITLE
fixing CMP GUI so it does not always react + bug fix

### PR DESCRIPTION
**Pull request by paulr34**

### CODE REVIEW

File: src/components/ZclEndpointCard.vue [hunk 1]
```diff
@@ -419,6 +419,15 @@ export default {
       if (this.$route.path !== '/') {
         this.$router.push({ path: '/' })
       }
+      if (this.$store.state.zap.isMultiConfig) {
+        if (categoryTmp === 'zigbee') {
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:423]: i think we should use helperCategory in this condition
dbEnum.helperCategory.matter
dbEnum.helperCategory.zigbee

File: src/components/ZclEndpointCard.vue [hunk 2]
```diff
@@ -491,11 +500,27 @@ export default {
         return this.$store.state.zap.endpointView.parentEndpointIdentifier
       }
     },
+    showParentEndpointIdentifier: {
+      get() {
+        return (
+          this.getDeviceCategory(this.deviceType[0].packageRef) === 'matter'
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:506]: same here

File: src/components/ZclEndpointCard.vue [hunk 3]
```diff
@@ -491,11 +500,27 @@ export default {
...
+    },
     profileId: {
       get() {
         return this.$store.state.zap.endpointView.profileId
       }
     },
+    showProfileId: {
+      get() {
+        return (
+          this.getDeviceCategory(this.deviceType[0].packageRef) === 'zigbee' &&
```
Comment (tbrkollar) [inline src/components/ZclEndpointCard.vue:518]: and here. dbEnum.helperCategory.zigbee

File: src/components/ZclEndpointCard.vue (hunk 1 above)
Comment (paulr34) [reply src/components/ZclEndpointCard.vue]: thank you, fixed

**Review by tbrkollar — APPROVED** [review]
if tests are passed

**Pull request merged**